scripts/utils/release-utils.ts

```diff
@@ -125,7 +125,14 @@ export async function postGithubRelease(opts: BuildOptions): Promise<void> {
   // https://docs.github.com/en/github/administering-a-repository/automation-for-release-forms-with-query-parameters
   const url = new URL(`https://github.com/${opts.ghRepoOrg}/${opts.ghRepoName}/releases/new`);
   url.searchParams.set('tag', versionTag);
-  url.searchParams.set('title', title);
+
+  const now = new Date();
+  let timestamp = now.getUTCFullYear() + '-';
+  timestamp += ('0' + (now.getUTCMonth() + 1)).slice(-2) + '-';
+  timestamp += ('0' + now.getUTCDate()).slice(-2);
+
+  url.searchParams.set('title', encodeURIComponent(`${title} (${timestamp})`));
+
```

Comment:
I put the following in a browser console:
```typescript
const title = '🌵v2.8.0-0';
const now = new Date();
let timestamp = now.getUTCFullYear() + '-';
timestamp += ('0' + (now.getUTCMonth() + 1)).slice(-2) + '-';
timestamp += ('0' + now.getUTCDate()).slice(-2);
console.log(encodeURIComponent(`${title} (${timestamp})`);
// %F0%9F%8C%B5v2.8.0-0%20(2021-09-16)
```
and navigated to the 'create a GitHub release' URL with `?title=THAT_VAL`